### PR TITLE
OpenAPI Spec: Add advanced parameters for retrieving results

### PR DESCRIPTION
**Pull request by RichardKeo**
Add the following parameters when retrieving results:
  - `columns`: columns to include in the results
  - `filters`: expression to filter out some rows from the results
  - `sort_by`: expression to specify the order in which to return the results

### CODE REVIEW

**Comment by TheEdgeOfRage** [comment]
btw, there already is a PR for the columns param here #42 I'm fine with closing that one in favor of this one, but I like Miguel's suggestion on adding the Tip about reducing datapoint costs

**Review by TheEdgeOfRage — APPROVED** [review]

File: api-reference/executions/exec-openapi.json [hunk 1]
```diff
@@ -377,7 +377,25 @@
...
+            "description": "Expression to filter out rows from the results to return. This expression is similar to a SQL WHERE clause. More details about it in the filters section below. This parameter is incompatible with sampling."
+          },
+          {
+            "in": "query",
+            "name": "sort_by",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Expressiong to define the order in which the results should be returned. This expression is similar to a SQL ORDER BY clause. More details about it in the Sort by section below."
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:398]: `Sort by` casing

File: api-reference/executions/exec-openapi.json [hunk 2]
```diff
@@ -534,7 +552,25 @@
...
+            "description": "Number of rows to return from the result by sampling the data. This is useful when you want to get a uniform sample instead of the entire result. If the result has less than the sample count, the entire result is returned. Note that this will return a randomized sample, so not every call will return the same result. This parameter is incompatible with pagination (limit and offset)."
+          },
+          {
+            "in": "query",
+            "name": "filters",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Expression to filter out rows from the results to return. This expression is similar to a SQL WHERE clause. More details about it in the filters section below. This parameter is incompatible with sampling."
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:564]: `filters` casing.

File: api-reference/executions/exec-openapi.json [hunk 3]
```diff
@@ -534,7 +552,25 @@
...
+          },
+          {
+            "in": "query",
+            "name": "sort_by",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Expressiong to define the order in which the results should be returned. This expression is similar to a SQL ORDER BY clause. More details about it in the Sort by section below."
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:573]: `Sort` casing

**Review by msf — COMMENTED** [review]
looks good. 
should we normalize casing of:
- `Sort by`
- `Filters`
- `Sample`

**Review by msf — APPROVED** [review]

**Pull request merged**